App/Navigation/PhotosNavigation.js

```diff
@@ -112,7 +112,7 @@ const TabNav = TabNavigator(
     tabBarComponent: TabBarBottom,
     tabBarPosition: 'bottom',
     animationEnabled: false,
-    swipeEnabled: false
```

Comment:
I had specifically set this to false. Any reason it needs to be true?

Reply:
oh sorry, that was a mistake... was playing around and didn't set it back. will revert